OpenSim/Common/TableSource.h

```diff
@@ -202,21 +202,30 @@ class TableSource_ : public Component {
                          TimeOutOfRange, 
                          time, timeCol.front(), timeCol.back());
 
-        const auto colInd = _table.getColumnIndex(columnLabel);
+        const auto colInd = 
+            static_cast<int>(_table.getColumnIndex(columnLabel));
```

Comment:
Out of curiosity, why does `_table.getColumnIndex()` return a size_t (unsigned) while `_storage.getSize()` returns an int (see the change in Storage.cpp, above)? It seems to be equally impossible for a size and a column index to be negative.